src/Main.cpp

```diff
@@ -317,6 +319,14 @@ void poll(Driver& driver, RTCMForwarder* rtcm_in, RTCMForwarder* rtcm_out) {
     }
 }
 
+DevicePort portFromString(string arg) {
```

Comment:
Is  `arg` usually small enough (is it a number?) that makes it ok passing by value instead of const ref?